src/css/timeline.css

```diff
@@ -0,0 +1,63 @@
+.tweetTable {
+	border: solid 1px;
+	width: 90%;
+	margin: 8px auto auto auto;
```

Comment:
simplify this rule

Reply:
Done, although it adds 8px to bottom margin